examples/server/routes/app.vue

```diff
@@ -0,0 +1,9 @@
+<template>
+  <NuxtExampleLayout example="server/routes">
+    <p>Fetched from <pre>/api/mountain</pre>: {{ mountain }}</p>
+  </NuxtExampleLayout>
+</template>
+
+<script setup>
+const { data: mountain } = await useAsyncData('mountain', () => $fetch('/api/mountain').catch(console.trace))
```

Comment:
Should we use `useFetch` here or is the aim to show a more complex use-case?

```suggestion
const { data: mountain } = await useFetch('/api/mountain')
```

Reply:
We can definitely use `useFetch` indeed, as the focus is more on making a server route.